.github/workflows/lint.yaml

```diff
@@ -0,0 +1,21 @@
+name: Validate bootstrap
```

Comment:
This sounds a bit too general. Maybe something more specific like `Validate generated YAML` would be more informative?